### PR TITLE
Fixes bug 1144169 add script for a stage refresh into a heroku database

### PR DESCRIPTION
**Pull request by selenamarie**
This includes a script to pull a minimal set of data from a stage database and put it into Heroku.

### CODE REVIEW

File: scripts/database/create_heroku_stage_database.sh [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+source /etc/socorro/socorrorc
+
+# Enable access to installed prod python modules
+source ${VENV}/bin/activate
+
+###############################################################################
+# END CONFIG
+###############################################################################
+
+echo date
```
Comment (phrawzty) [inline scripts/database/create_heroku_stage_database.sh]: timestamping at intervals :+1:

File: scripts/database/create_heroku_stage_database.sh [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        --no-acl --no-owner \
+        --disable-triggers \
+        -v \
+        -f ${SLIM_DUMP} \
+        ${SOURCE_DATABASE}
+
+echo date 
+
+# Move directories so that we get the alembic environment configuration
+pushd /data/socorro/application
```
Comment (phrawzty) [inline scripts/database/create_heroku_stage_database.sh]: :+1: pushd / popd model

File: scripts/database/create_heroku_stage_database.sh [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+# Setup a fresh database from our repo
+# The hostname is set to '' so that psycopg2 uses a local socket, thereby avoiding ACLs
+$PYTHON ${VENV}/bin/socorro setupdb --database_name=${TEMP_DATABASE} --no_staticdata --dropdb --no_roles\
```
Comment (phrawzty) [inline scripts/database/create_heroku_stage_database.sh]: Minor nit, but why not put each arg on its own line?

File: scripts/database/create_heroku_stage_database.sh [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+# Reset our stage database
+heroku pg:reset ${HEROKU_STAGE_APP} --confirm ${HEROKU_STAGE_APP}
+
+# Restore to heroku
+pg_restore --no-owner --no-acl -v -d ${HEROKU_STAGE_DB_URL} ${FULL_DUMP}
+
+# Cleanup our temp database
+$PSQL postgres -c "DROP DATABASE ${TEMP_DATABASE}"
+
+
```
Comment (phrawzty) [inline scripts/database/create_heroku_stage_database.sh]: This script doesn't appear to have any explicit error handling, nor is it `set -e` - is there any chance that :racehorse: through these commands could have adverse side effects if one fails?

File: scripts/database/create_heroku_stage_database.sh (hunk 3 above)
Comment (selenamarie) [inline scripts/database/create_heroku_stage_database.sh]: Oh! I can separate them. No problem :)

File: scripts/database/create_heroku_stage_database.sh (hunk 4 above)
Comment (selenamarie) [inline scripts/database/create_heroku_stage_database.sh]: Yeah, we should have set -e in there. I had an issue where I set it and then an error got swallowed though. I will investigate more closely and get `set -e` back.

**Comment by phrawzty** [comment]
`r+`

**Pull request merged**